packages/automerge-repo-storage-memory/test/index.test.ts

```diff
@@ -0,0 +1,5 @@
+import { expect } from "chai"
```

Comment:
let's move this into a test helper -- you should never use this in real life. if you don't want to actually store things: you don't have to store them! (just initialize the object without storage)

Reply:
✅